### PR TITLE
Fixes #51: Subclasses can override #can_crud?

### LINKED ISSUE #51: Update policy defaults so that there is a protected can_crud? method for crud actions that can be overridden in sub classes
```ruby
  alias :index?   :can_crud?
  alias :new?     :can_crud?
  alias :create?  :can_crud?
  alias :edit?    :can_crud?
  alias :update?  :can_crud?
  alias :destroy? :can_crud?

protected

  def can_crud?
    is_admin? || is_member?
  end
```

SUBCLASS

```ruby
protected

  def can_crud?
    # some other logic right up in here
  end
```

### PR DESCRIPTION
**Pull request by jordanmaguire**
This allows developers to change the behaviour of all the CRUD authorization checks in one place in subclasses.

### CODE REVIEW

**Comment by psyren89** [comment]
:+1:

File: app/policies/application_policy.rb [hunk 1]
```diff
@@ -25,11 +25,34 @@ def is_member?
...
+
+  def destroy?
+    can_crud?
+  end
+
+protected
+
+  def can_crud?
+    is_admin?
+  end
```
Comment (swebb) [inline app/policies/application_policy.rb]: Would be nice to have a shared example that matches up to can_crud? That way we can use it for anything that doesn't override it.

**Comment by swebb** [comment]
:+1:

**Comment by osiro** [comment]
:+1:

File: app/policies/application_policy.rb (hunk 1 above)
Comment (jordanmaguire) [inline app/policies/application_policy.rb]: There is one already :+1:

**Pull request merged**